cdap-client/pom.xml

```diff
@@ -112,6 +112,10 @@
       <artifactId>cdap-explore-client</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
So if someone depends on cdap proto and uses ProgramRunId class, they also have to declare this dependency?